### PR TITLE
Consolidates all work done on individual CR* branches

### PR DESCRIPTION
**Pull request by danielkhan**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/pull/246.html" title="Last updated on Feb 26, 2019, 4:49 PM UTC (02adfa9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/246/3e0c447...02adfa9.html" title="Last updated on Feb 26, 2019, 4:49 PM UTC (02adfa9)">Diff</a>

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@danielkhan can you please comment on the need for this PR? I see unresolved comments on individual PRs still there. Like here: https://github.com/w3c/trace-context/pull/222/files Do you envision all discissions moved into this PR?

**Comment by danielkhan** [comment]
@SergeyKanzhelev The need is to prepare the spec for CR. As such I propose to merge the work done in various other branches into master.
For ongoing discussions, as they seem to have stalled let's go through them on our meeting on Tuesday where we should then reach a level of agreement that allows us to do some last edits and merge.

Alternatively we could also create a release candidate branch but I feel that there are enough branches already and a merge is due.

**Comment by AloisReitbauer** [comment]
Agreed that we need to consolidate all edits soon for an updated CR version. The current master is quite outdated already.  Let us work on merging as much as we can. 

For everything that needs to be discussed, we should create issues.

**Comment by SergeyKanzhelev** [comment]
> The current master is quite outdated already.

This is very strong statement. Master reflects current state of specification and existing implementations following it. There is no proposed and approved change that HAVE TO be made.

I agree, however, some things needs improvement and final discussions. Those are tracked in 13 issue left in CR milestone: https://github.com/w3c/trace-context/milestone/5

> Let us work on merging as much as we can.

I'd argue merging smaller updates is easier. I'd really love to work on and merge [abstract update](https://github.com/w3c/trace-context/pull/221) to address #203, but it may not be the best idea to block this merge on discussion of processing model that potentially may spill over to the next milestone as it's mostly clarifications and reformatting of a spec.

I'm open for opinions, but not sure how to review this PR. Should all the comments from smaller PR be moved here? Were there any changes comparing to those small PRs?

**Comment by danielkhan** [comment]
The contents of this PR are the processing model plus a lot of editorial changes/language fixes that have already been discussed. There should be no semantic changes included but I missed that there was still an ongoing discussion around  https://github.com/w3c/trace-context/pull/222/files. Do you think that we can come to a conclusion on that within the next meeting @SergeyKanzhelev?

The PR is that large because of the many language fixes that can hardly be split up into smaller PRs.

As this PR includes commits from smaller PRs the changes are included and there should not be any changes beyond what has been discussed already.

**Review by AloisReitbauer — APPROVED** [review]

**Comment by AloisReitbauer** [comment]
Can we agree to get the changes merged as we addressed all them or added an issue where further discussion is needed

**Comment by SergeyKanzhelev** [comment]
@AloisReitbauer I'd suggest not to merge anything marked with "PENDING DECISION". It's better to allow both behavior in case there is something not defined in spec than have this text.

My understanding also was that during the last meeting you discussed that all language and non-required changes can be merged in PR timeframe in order to minimize randomization now and close on CR.

**Comment by danielkhan** [comment]
> @AloisReitbauer I'd suggest not to merge anything marked with "PENDING DECISION". It's better to allow both behavior in case there is something not defined in spec than have this text.

Removed 'PENDING DECISION' remarks in favor of their respektive issues.

File: spec/30-processing-model.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+As stated above, tracing systems can decide for each request whether they want to
+just _forward_ or _participate_ in a trace context. Additionally, when participating,
+a system needs to be able to start or restart a trace.
+
+## Processing Model in Forwarding Mode
+
+In forwarding mode, a tracing system decides to forward tracing information
+but does not modify the tracecontext headers.
+
+1. The tracing system receives a `traceparent` header and may receive a `tracecontext`.
```
Comment (discostu105) [inline spec/30-processing-model.md:13]: You wrote `tracecontext`, but don't you mean `tracestate`? There are multiple such occurrences in the following text.

**Comment by danielkhan** [comment]
Closed in favor of https://github.com/w3c/trace-context/pull/259

**Pull request closed without merging**